crates/agentgateway/src/http/transformation_cel.rs

```diff
@@ -249,39 +207,9 @@ impl<'a> RequestOrResponse<'a> {
 			(Some(v), _) => {
 				if let RequestOrResponse::Request(r) = self
 					&& let Some(b) = cel::value_as_bytes(&v)
+					&& !crate::http::apply_pseudo_to_request(r, k, b)
```

Comment:
do we want to have the `RequestOrResponse` concept from transformation brought up, and then we don't need the `request` and `response` variants?